pandas/tests/test_resample.py

```diff
@@ -649,13 +649,6 @@ def test_asfreq_fill_value(self):
         expected = frame.reindex(new_index, fill_value=4.0)
         assert_frame_equal(result, expected)
 
-    def test_resample_interpolate(self):
```

Comment:
Removed this test as it was failing - investigated and seems that this test is from a closed PR #12974 opened for issue #12925
Not sure if this is the right call....
probably there are other tests failing which I haven't investigated yet - my sense is those might be related to this one - will check the TravisCI and other checks for it again.

Reply:
Restored this test now - this test along with others were failing due to error in interpolation, which is fixed now